### PR TITLE
Ft landing page

### PR DESCRIPTION
**Pull request by fab-ryan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 #### Installation
 
 - npm i or npm install if your are using npm package manager and if you are using yarn, use type yarn only.
+- make copy of .env.example to .env and make change due to file configuration
```
Comment (Anguandia) [inline README.md:6]: Correct the grammer

**Review by Anguandia — DISMISSED** [review]

**Review by GilbertUp — APPROVED** [review]

**Review by gerard1001 — APPROVED** [review]

File: src/style.css [hunk 2]
```diff
@@ -1,5 +1,42 @@
-@import "./css/Logout.css"; /* Style for Logout page */
+@import url("https://fonts.googleapis.com/css?family=Inter");
```
Comment (furebo) [inline src/style.css:2]: Resolve the conflicts in this file src/style.css

File: src/pages/Logout.jsx [hunk 3]
```diff
@@ -8,6 +8,7 @@ import notify from '../images/notify.png'
 import logout from '../images/logout.png'
```
Comment (furebo) [inline src/pages/Logout.jsx:8]: Resolve conflicts in this file src/pages/Logout.jsx

**Review by furebo — CHANGES_REQUESTED** [review]
Reduce the number of commits (squash) resolve the conflicts and delete the commented codes  in logout.jsx file

**Review by furebo — APPROVED** [review]

**Pull request merged**